packages/hooks/src/useForceRerender.tsx

```diff
@@ -0,0 +1,11 @@
+import { useCallback, useState } from 'react';
+
+/**
+ * Hook that forces a re-render.
+ */
+export function useForceRerender(): () => void {
+  const [, updateState] = useState<Object>({});
+  const forceRerender = useCallback(() => updateState({}), []);
+
+  return forceRerender;
+}
```

Comment:
Nit: I think we're trying to put each hook in its own folder so we can collocate tests